docs/_sidebar.md

```diff
@@ -3,7 +3,8 @@
   - [Configure](/ops/configuring.md)
   - [UI](/ui.md#armadillo-user-interface "Armadillo 3 UI")
   - [FAQ](/faq.md)
-  - [Armadillo 2 to 3 migration guide](/migration-guide-2-to-3.md#instructions-for-migrating-a-2x-armadillo-to-3x "Armadillo 2 to 3 migration guide")
+  - [Migration from 2 to 3](/upgrade-2-3.md)
```

Comment:
Could you also change this reference in [_navbar.md](https://raw.githubusercontent.com/molgenis/molgenis-service-armadillo/master/docs/_navbar.md)?